### ISSUE TITLE
Allow to specifiy context in k8s client

### ISSUE THREAD

**Issue by teoyaomiqui** [airshipit/airshipctl#388]
**Problem description (if applicable)**
We currently relay on specifying kubeconfig context when running phases, however if we will need to use k8s client integrated in airshipctl, it does not support context, and uses only current context.

**Proposed change**
Change k8s client to accept context as parameter
- [link to k8s client constructor](https://github.com/airshipit/airshipctl/blob/master/pkg/k8s/client/client.go#L63
)
- [first link to where this context should be used](https://github.com/airshipit/airshipctl/blob/master/pkg/k8s/client/client.go#L68)

**Comment by teoyaomiqui** [comment]
**Note**
In parallel we should make all commands that use kubeconfig to be context aware as well, should we create a separate issue for that, or extend this one?

**Comment by teoyaomiqui** [comment]
@sreejithpunnapuzha I have made this patchset before creating an issue, you can use it as a reference, if needed https://review.opendev.org/#/c/761232/

**Comment by mf4716** [comment]
@sreejithpunnapuzha - Sreejith, are we close(r) to finishing this issue?

**Comment by teoyaomiqui** [comment]
@mf4716 we can close this issue, it was fixed some time ago
